.github/workflows/publish.yml

```
@@ -21,3 +21,28 @@ jobs:
       uses: JS-DevTools/npm-publish@v1
       with:
         token: ${{ secrets.NPM_TOKEN }}
+  github:
+    name: GitHub release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Get release version
+      id: get_version
+      run: |
+        export PACKAGE_VERSION=$(cat package.json | grep 'version' | sed 's/[ \",:]//g' | sed 's/version//')
+        echo "::set-output name=version::$PACKAGE_VERSION"
+    - name: Create and push git tag
+      uses: actions-ecosystem/action-push-tag@v1
+      with:
+        tag: ${{ steps.get_version.outputs.version }}
+        message: ${{ steps.commit.outputs.git-message }}
+    - name: Create release
+      uses: actions/create-release@v1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ steps.get_version.outputs.version }}
+        release_name: Release ${{ steps.get_version.outputs.version }}
```

OK, since we are going to take the opportunity to update the workflow of the main repository, do you have any preference in using underscores here?

```suggestion
      id: get-version
      run: |
        export PACKAGE_VERSION=$(cat package.json | grep 'version' | sed 's/[ \",:]//g' | sed 's/version//')
        echo "::set-output name=version::$PACKAGE_VERSION"
    - name: Create and push git tag
      uses: actions-ecosystem/action-push-tag@v1
      with:
        tag: ${{ steps.get-version.outputs.version }}
        message: ${{ steps.commit.outputs.git-message }}
    - name: Create release
      uses: actions/create-release@v1
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      with:
        tag_name: ${{ steps.get-version.outputs.version }}
        release_name: Release ${{ steps.get-version.outputs.version }}
```

---

No preference whatsoever, let's change it :+1: